File: pickem-web/src/app/sub-system/models/api/player-scoreboard.ts
import { GameScoreboard } from './game-scoreboard';

export class PlayerScoreboard
{
    games: number;
    gamesPicked: number;
    gamesWon: number;
    gamesLost: number;
    gamesPending: number;
    points: number;

    gamePickScoreboards: GameScoreboard[];

    constructor() { this.gamePickScoreboards = []; }
}

File: pickem-bots/ExampleCSharpBot/ExampleCSharpBot/PickemModels/PlayerScoreboard.cs
using System.Collections.Generic;

namespace ExampleCSharpBot.PickemModels
{
    public class PlayerScoreboard
    {
        public int Games { get; set; }
        public int GamesPicked { get; set; }
        public int GamesWon { get; set; }
        public int GamesLost { get; set; }
        public int GamesPending { get; set; }
        public int Points { get; set; }

        public List<GameScoreboard> GamePickScoreboards { get; set; }
    }
}
